index.js

```diff
@@ -1,16 +1,64 @@
+/* jshint node: true */
+/* jshint asi: true */
 var connect = require('connect')
 
 var config = require('./config.json')
 
-function helmet () {
+function findModuleName(alias) {
+  'use strict';
+  var result = null;
+  Object.keys(config.middlewares).forEach(function (moduleName) {
+    if (alias === moduleName || config.middlewares[moduleName].indexOf(alias) > -1) {
+      result = moduleName
+    }
+  })
+  return result
+}
+
+function helmet (optionOverrides) {
+  'use strict';
   var chain = connect()
+
+  // init options for default middleware
+  var options = {}
   config.defaultMiddleware.forEach(function (middlewareName) {
-    chain.use(helmet[middlewareName]())
+    var moduleName = findModuleName(middlewareName)
+    // TODO: initialize with real default props from module?
+    options[moduleName] = {}
+  })
+
+  // override options with settings provided externally
+  Object.keys(optionOverrides).forEach(function (middlewareName) {
+    var overrides = optionOverrides[middlewareName]
+    var moduleName = findModuleName(middlewareName)
+    // console.log(middlewareName)
+    // console.log(moduleName)
+    // console.log(overrides)
+    if (typeof overrides === 'boolean') {
+      if (overrides) {
+        // TODO: initialize with real default props from module?
+        options[moduleName] = {}
+      } else {
+        delete options[moduleName]
+      }
+    } else {
+      options[moduleName] = overrides
+    }
+  })
+
+  // build chain of helmet modules
+  Object.keys(options).forEach(function (moduleName) {
+    chain.use(helmet[moduleName](
+      // don't pass in empty object
+      Object.keys(options[moduleName]).length ? options[moduleName] : undefined
+    ))
```

Comment:
This works for all modules, except frameguard which takes params instead of an options object. Filed a ticket: https://github.com/helmetjs/frameguard/issues/7